addons/rose/tests/integration/components/rose/code-editor-test.js

```diff
@@ -0,0 +1,27 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | rose/code-editor', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    let codeValue = 'export HCP_CLIENT_ID = {place your client_id here}';
```

Comment:
Not blocking:
Can `codeValue` be a `const` ?  Seems to me the variable does not change its state.

Reply:
Yup good call, I updated the places I used `let`!